integration_test/common.js

```diff
@@ -307,11 +324,88 @@ module.exports = {
     return [app1, app2];
   },
 
+  async addFriendToNewClosedGroup(app, app2) {
+    await app.client
+      .element(ConversationPage.closedGroupNameTextarea)
+      .setValue(this.VALID_CLOSED_GROUP_NAME1);
+    await app.client
+      .element(ConversationPage.closedGroupNameTextarea)
+      .getValue()
+      .should.eventually.equal(this.VALID_CLOSED_GROUP_NAME1);
+
+    await app.client
+      .element(ConversationPage.createClosedGroupMemberItem)
+      .isVisible();
```

Comment:
You have to test the returned value, otherwise I think it does not perform any checks. 
Some of my calls are missing this check too in fact. (moved from `waitForExists` which fails if not found after timeout to `isVisible` or `isExisting` which just returns the value and forgot to update some checks). I can update mine once this one is merged.
Something like `.should.eventually.be.true`